nationbuilder-rb.gemspec

```diff
@@ -1,12 +1,8 @@
-# Generated by jeweler
-# DO NOT EDIT THIS FILE DIRECTLY
-# Instead, edit Jeweler::Tasks in Rakefile, and run 'rake gemspec'
-# -*- encoding: utf-8 -*-
-# stub: nationbuilder-rb 1.4.2 ruby lib
+require './lib/nationbuilder/version'
 
 Gem::Specification.new do |s|
   s.name = "nationbuilder-rb"
-  s.version = "1.4.3"
+  s.version = NationBuilder::VERSION
```

Comment:
Why is it better to set the version number in `NationBuilder::Version` instead of in the gemspec?